nix/.stack.nix/default.nix

```diff
@@ -50,6 +50,7 @@
         cardano-prelude = ./cardano-prelude.nix;
         cardano-prelude-test = ./cardano-prelude-test.nix;
         cardano-crypto = ./cardano-crypto.nix;
+        cardano-crypto = ./cardano-crypto.nix;
```

Comment:
Is this change needed?

Reply:
You are reviewing nix files now..? :D It was a change produced by the stack2nix script from CI in https://github.com/input-output-hk/ouroboros-network/pull/1380 :man_shrugging:

Reply:
Fixed in #1380.